recipes/configs/llama2_eleuther_eval.yaml

```diff
@@ -0,0 +1,30 @@
+# Config for EleutherEvalRecipe in eluether_eval.py
+#
+# To launch, run the following command from root torchtune directory:
+#    tune eleuther_eval --config llama2_eleuther_eval
```

Comment:
Might even add tasks arg here explicitly given that (a) it's the most commonly used, and (b) list + CLI is annoying